docs/source/running-jobs/distributed-jobs.rst

```diff
@@ -187,31 +189,23 @@ To execute a distributed Ray program on many nodes, you can download the `traini
     workdir: .
 
     setup: |
-      conda activate ray
```

Comment:
we can add this `:emphasize-lines: 26-28` (check the line number) at the top to emphasize the `~/sky_templates/ray/start_cluster.sh` line